controller/stats.controller.js

```diff
@@ -4,6 +4,8 @@ const ServerError = require("../errors/internal.errors");
 const ExperimentRepository = require("../repositories/experiment.repository");
 const UserRepository = require("../repositories/user.repository");
 const {mongoose} = require("mongoose");
+const Experiment = require("../models/Experiment.model");
```

Comment:
same as before it is advised to creat a method in repository and only use the repository